src/backend/TrafficCourts/Citizen.Service/Features/Disputes/Create.cs

```diff
@@ -112,16 +117,22 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                             // Serialize OCR violation ticket to a JSON string
                             ocrViolationTicketJson = JsonSerializer.Serialize(violationTicket);
                         }
+                        // Get the looked up ticket
                         else
                         {
                             // Get the looked up violation ticket data from Redis cache using the ticket id key
                             lookedUpViolationTicket = await _redisCacheService.GetRecordAsync<Models.Tickets.ViolationTicket>(ticketId);
+
+                            if (lookedUpViolationTicket is null)
+                            {
+                                _logger.LogError("No looked up violation ticket has been found for the {ticketId}", ticketId);
```

Comment:
Can we make properties we log using Pascal Casing. Having consistent property names across the systems make reporting on the logs easier.

Reply:
Which property do you suggest changing into Pascal Case?